src/wiseagents/vectordb/lang_chain_wise_agent_vector_db.py

```diff
@@ -20,7 +21,19 @@ def __new__(cls, *args, **kwargs):
         """Create a new instance of the class, setting default values for the instance variables."""
         obj = super().__new__(cls)
         obj._embedding_model_name = DEFAULT_EMBEDDING_MODEL_NAME
-        obj._embedding_function = HuggingFaceEmbeddings(model_name=DEFAULT_EMBEDDING_MODEL_NAME)
+        try:
+            import sentence_transformers  # type: ignore[import]
+
+        except ImportError as exc:
+            raise ImportError(
+                "Could not import sentence_transformers python package. "
+                "Please install it with `pip install sentence-transformers`."
```

Comment:
Is this needed? `sentence-transformers` is in the pyproject.toml file.

Reply:
Nope